packages/itwinui-css/src/button/button.scss

```diff
@@ -33,42 +32,34 @@
   }
 
   &[data-iui-active='true'] {
-    @include default.iui-button-default-active;
+    @include base.iui-button-active;
 
     &[data-iui-variant='borderless'] {
       @include borderless.iui-button-borderless-active;
     }
-
-    @include default.iui-button-active-disabled;
   }
 
-  &[data-iui-variant='high-visibility'] {
-    @include variant.iui-button-variant(high-visibility);
-  }
+  @include variant.iui-button-variant;
 
-  &[data-iui-variant='cta'] {
-    @include variant.iui-button-variant(cta);
-  }
-
-  &[data-iui-variant='idea'] {
+  &[data-iui-type='feedback'] {
     @include idea.iui-button-idea;
   }
-}
 
-.iui-button-icon {
-  @include base.iui-button-icon;
-}
+  &[data-iui-type='dropdown'] {
```

Comment:
I'm thinking this one should stay as a class? `iui-button-dropdown`

Similarly, the ideas button could also be a class: `iui-button-idea`

Multiple reasons:
- The `data-iui-type` attribute feels a little strange; the name is too vague.
- "dropdown" and "idea" have nothing to do with each other. They affect totally different CSS properties.
- In React, `DropdownButton` and `IdeasButton` are their own components.

Reply:
The thought was using a data attribute makes it a binary choice.  You can't combine a dropdown and an ideas button.  If split button wasn't actually a combination of 2 buttons but instead was 1 button, it would fall under this as well.

Reply:
That's not a strong enough argument, especially when you look at the other reasons I listed. You also can't combine a Table and a Tile.